tests/models/Feed_test.js

```diff
@@ -46,7 +46,17 @@ global.getTestDB = co.wrap(function* () {
   return sequelize;
 });
 
-describe('Feed', function(){
+global.getFeed = co.wrap(function* (){
+	return{
+		id: 1,
+		owner_uid: 1,
+		title: 'Test Feed',
+		feed_url: 'Test Feed'
+	};
+});
```

Comment:
This function is synchronous, therefore there is no need to use `co.wrap` or `yield` when calling the function.